__tests__/app.test.js

```diff
@@ -115,4 +117,59 @@ const { convertTimestampToDate } = require("../db/seeds/utils.js");
                         });
         })
     })
+    describe("GET/api/articles/:article_id/comments", () => {
+        test("status:200 - responds with an array of comments for the given article id, with the correct properties", () => {
+            return request(app)
+                .get("/api/articles/1/comments")
+                .expect(200)
+                .then(({ body }) => {
+                    const { comments } = body;
+                    
+                    expect(comments).toBeInstanceOf(Array);
+                    expect(comments).toHaveLength(11);
+                    comments.forEach((comment) => {
+                       
+                        expect(comment).toMatchObject({
+                            comment_id: expect.any(Number),
+                            votes: expect.any(Number),
+                            created_at: expect.any(String),
+                             author: expect.any(String),
+                           body: expect.any(String),
+                           article_id:expect.any(Number),
```

Comment:
Its a small one, but would ideally want to make sure the id is 1 - at the moment, you could be getting the wrong one and wouldnt really know
`article_id: 1,`